src/audio/asrc/asrc.c

```diff
@@ -367,13 +367,84 @@ static struct comp_dev *asrc_new(const struct comp_driver *drv,
 }
 #endif
 
+static int asrc_initialise_buffers(struct asrc_farrow *src_obj)
```

Comment:
I see that it was like this before and I might be guilty myself too for letting some of these slip through (or thru :-) ) too, but I think we're trying to use the US spelling throughout the sources, so we could as well use this opportunity to make this `initialize`